### DISCUSSION TITLE
Can you add touch controls?

### DISCUSSION THREAD

**Discussion by starqus** [DroidDrone/DroidDrone#7] [Ideas]
It is difficult to use with joystick. Can you add control buttons on the screen?  Also will there be a way to connect to the Qground control app or Will the comtrol application turn into such an gcs application?

**Comment by IvanSchulz** [comment]
Hi, I currently have less experience with QGroundControl, can look into it in the future. What kind of joystick do you use? I connect my Taranis X7 to the phone and can control it like with a "normal" receiver.

**Comment by IvanSchulz** [comment]
With the version 2.3.0 you can now connect the QGroundControl ;)
